src/flow-spec/IBlockExit.ts

```diff
@@ -78,7 +78,3 @@ export interface IBlockExit {
   // todo: we need to update docs -- they specify "key presence", but I'd prefer us to be more explicit
   default?: boolean
```

Comment:
Does our backend use this property? I've tried a tree with some blocks and the exit looked like this:
```
{
  "uuid": "29841127-3247-4610-8000-d26831becf82",
  "label": "29841127-3247-4610-8000-d26831becf82",
  "tag": "Default",
  "destination_block": null,
  "test": "@(TRUE)",
  "config": {}
}
```

Reply:
More on the backend changes:
The default exit should probably have `"test": "@(FALSE)"` or simply miss one (undefined) to avoid confusion with regular exits. See the logic in `findDefaultBlockExitOrThrow()` and `_firstBlockExit()`.
Plus the default exit should have `"default": true`.